src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -221,14 +275,16 @@ public void initializeFromEntity(@NotNull final EntityCitizen entity)
         maxHealth = entity.getMaxHealth();
         experience = 0;
         level = 0;
+        saturation = 10;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Assign this magic number 10 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)